src/main/java/org/broadinstitute/hellbender/tools/CountReads.java

```diff
@@ -39,6 +39,7 @@ public void apply( final GATKRead read, final ReferenceContext referenceContext,
 
     @Override
     public Object onTraversalSuccess() {
+        logger.info("CountReads counted " + count + " total reads");
```

Comment:
@droazen This should probably print a non-logger number directly to standard out as well shouldn't it?  So that people can capture it programmatically/ incase logging is turned off.

Reply:
This PR is just to label the value output by the tool, which was confusing users. If we want to support actual piping of the output, which is not something we typically do, we'd probably want an actual test case to prove that it works. This seems beyond the scope of this particular PR.